nvflare/edge/executors/et_edge_model_executor.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from typing import Optional
+
+import torch
+from torch import nn
+
+from nvflare.apis.dxo import DataKind, from_dict
+from nvflare.apis.fl_context import FLContext
+from nvflare.apis.shareable import ReservedHeaderKey
+from nvflare.edge.constants import CookieKey, MsgKey
+from nvflare.edge.executors.edge_model_executor import EdgeModelExecutor, ModelUpdate
+from nvflare.edge.executors.hug import TaskInfo
+from nvflare.edge.model_protocol import ModelBufferType, ModelEncoding, ModelExchangeFormat, ModelNativeFormat
+from nvflare.edge.models.model import export_model_to_bytes
+from nvflare.edge.mud import BaseState
+from nvflare.edge.web.models.result_report import ResultReport
+
+
+class ETEdgeModelExecutor(EdgeModelExecutor):
+    def __init__(
+        self,
+        et_model: nn.Module,
```

Comment:
Should be nvflare.edge.models.model.DeviceModel.

Reply:
changed.